docs/licensing.md

```diff
@@ -0,0 +1,2 @@
+!!! warning
```

Comment:
Remove this and just specify the license linking to LICENSE in the repo. That's all.